Frends.Template/setup.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+dir="../.github/workflows"
+
+if [[ ! -f "../LICENSE" ]]; 
+then
+	mv "./LICENSE" "../LICENSE"
+else
+	rm "./LICENSE"
+fi
+
+if [[ ! -d $dir ]];
+then
+	mkdir -p $dir;
+fi
+
+cd "workflows"
+
+for filename in *; do
+	echo $filename
+	mv "$filename" "../../.github/workflows";
+done
+
+cd ..
+
+echo "workflow files moved."
+
+rmdir "workflows"
+
+rm "setup.cmd"
+
+rm setup.sh
```

Comment:
Siis... mun mielestä se ei osaannutkaan poistaa ne setup tiedostot. Toimiiko se sulla?

Reply:
Did you get any errors when using the template. I tested with Linux ubuntu OS and everything just works likewise with windows. So there might be something with macOS that the scripts can't handle correctly.